cmd2.py

```diff
@@ -100,6 +129,44 @@
     except ImportError:
         pass
 
+
+# Tells what implementation of readline we are using
+class RlType(Enum):
```

Comment:
Need to make some sort of change so that this works on Python 2.7.  See the comment above in the imports regarding the ``enum`` module.